sdk/textanalytics/azure-ai-textanalytics/README.md

```diff
@@ -358,8 +358,45 @@ The azure-core [reference documentation](https://azuresdkdocs.blob.core.windows.
 describes available configurations for retries, logging, transport protocols, and more.
 
```

Comment:
We are not making any updates to the authentication section?

Reply:
Mariana posted on the channel that the auth info is fine for now. Not going to make any changes until we know what the plan is.

Reply:
I didn't realize we had different information on AAD in our repos. .NET has https://github.com/Azure/azure-sdk-for-net/tree/master/sdk/textanalytics/Azure.AI.TextAnalytics#create-textanalyticsclient-with-azure-active-directory-credential

I think it looked smaller which is why it was ok, although I did tell her I was going to add more information.

Reply:
I talked briefly with @annelo-msft and I still need to make changes to the .NET one to make sure people understand how to use AAD.

Reply:
I guess what I'm trying to say is that don't base your changes on .NET.
If we need to expand auth info in other repos we should do that (like with a link to a production env documentation)

Reply:
It looks like we have almost identical information, the only difference is that I formatted mine with bullet points making it look longer

Reply:
Didn't see your above comment; Maggie said she'd get back to me with guidance. I think a separate doc is a good idea, still kind of debating where to put it / what to call it.